<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QW_Panel_Chart2DClass</class>
 <widget class="QWidget" name="QW_Panel_Chart2DClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart2D Panel</string>
  </property>
  <widget class="QComboBox" name="comboBox_xAxis">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_xAxis">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>x Axis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plot">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_yAxis_01">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_yAxis_01">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y Axis</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_yAxis_02">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_yAxis_03">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_yAxis_04">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_yAxis_05">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_yAxis_02">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yAxis_03">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yAxis_04">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yAxis_05">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y Axis</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_01">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_02">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_03">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_04">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_05">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_01</sender>
   <signal>clicked()</signal>
   <receiver>QW_Panel_Chart2DClass</receiver>
   <slot>enable_combo_01()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_02</sender>
   <signal>clicked()</signal>
   <receiver>QW_Panel_Chart2DClass</receiver>
   <slot>enable_combo_02()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_03</sender>
   <signal>clicked()</signal>
   <receiver>QW_Panel_Chart2DClass</receiver>
   <slot>enable_combo_03()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_04</sender>
   <signal>clicked()</signal>
   <receiver>QW_Panel_Chart2DClass</receiver>
   <slot>enable_combo_04()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_05</sender>
   <signal>clicked()</signal>
   <receiver>QW_Panel_Chart2DClass</receiver>
   <slot>enable_combo_05()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_plot</sender>
   <signal>clicked()</signal>
   <receiver>QW_Panel_Chart2DClass</receiver>
   <slot>plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>plot()</slot>
  <slot>enable_combo_01()</slot>
  <slot>enable_combo_02()</slot>
  <slot>enable_combo_03()</slot>
  <slot>enable_combo_04()</slot>
  <slot>enable_combo_05()</slot>
 </slots>
</ui>
